jupyterlab/serverextension.py

```diff
@@ -8,6 +10,22 @@ def load_jupyter_server_extension(serverapp):
     """Temporary server extension shim when using
     old notebook server.
     """
+    serverapp.log.warning("Loading JupyterLab as an old notebook extension.")
```

Comment:
Maybe something more explicit like the following?

```suggestion
    serverapp.log.warning("Loading BrowserApp as a classic notebook (v6) extension.")
```

Reply:
To disambiguate the word "old".

Reply:
Thanks - just a comment this is not the `BrowserApp`, this is `JupyterLab` server extension hook.